soh/soh/GameMenuBar.cpp

```diff
@@ -781,6 +781,8 @@ namespace GameMenuBar {
                     UIWidgets::Tooltip("Wearing the Bunny Hood grants a speed increase like in Majora's Mask");
                     UIWidgets::PaddedEnhancementCheckbox("Mask Select in Inventory", "gMaskSelect", true, false);
                     UIWidgets::Tooltip("After completing the mask trading sub-quest, press A and any direction on the mask slot to change masks");
+                    UIWidgets::PaddedEnhancementCheckbox("Use Items Directly From Inventory", "gItemUseFromInventory", true, false);
```

Comment:
Can you set this to "off" in the Default presets as well?  Relevant function starts on line 97.

Reply:
Good call! Done 👍